packages/addons/service/spotify-connect-web/source/resources/settings.xml

```diff
@@ -3,7 +3,8 @@
    <category   label="30100" >
       <setting label="30101" type="lsep" />
       <setting label="30102" type="text"      id="scw_u" />
-      <setting label="30103" type="text"      id="scw_p" option="hidden" />
+      <setting label="30103" type="text"      id="scw_p" option="hidden"
+         visible="!eq(-1,)" />
```

Comment:
it is consistently badly formatted then .. something to clean-up in a future update :)